docs/releases/upgrading/upgrade-v12.0.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading from v11 to v12
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v11, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v11, you can gradually upgrade using the existing instructions to v11 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli`, for InvenioRDM v12 the release is v1.0.20.
+
+    ```bash
+    $ invenio-cli --version
+    invenio-cli, version 1.0.20
+    ```
+
+!!! info "Virtual environments"
+    In case you are not inside a virtual environment, make sure that you prefix each `invenio` command with `pipenv run`.
+
+### Installing the Latest Versions
+
+Bump the RDM version and rebuild the assets:
+
+```bash
+invenio-cli packages update 12.0.0
+```
+
+These commands should take care of locking the dependencies for v12, installing them, and building the required assets.
+
+### Database chema upgrade
+
+Run the database schema upgrade:
+
+```bash
+# Perform the database upgrade
+pipenv run invenio alembic upgrade
+```
+
+### Initialize new queues
+
+Initialize the new queues with:
+
+```bash
+# Perform the initialization of the new queues
+pipenv run invenio queues declare
+```
+
+### Update OS index
```

Comment:
minor: "OS" might be ambiguous, I suggest writing  "Open search" instead of "OS"